### DISCUSSION TITLE
Rename branch 'master' to 'main' in projects.

### DISCUSSION THREAD

**Discussion by TavoNiievez** [Codeception/Codeception#6161] [Ideas]
There are ways to make this change without breaking any projects.

The reasons for doing this are well documented on the Internet, but I will summarize that 'master' is an unfortunate term. 'main' means the same, it is shorter and it is more inclusive.

**Comment by DavertMik** [comment]
We dont need Master branch at all at Codeception, the development happens in Major version branch (4.x,5.x).
So we can even delete it. 
The only problem is how it will affect site geberation scripts, i hope we dont have it hardcoded...

Unfirtunately, i cant say it will be easy to update all other repos inside Codeception organization without breaking things. So lets postpone this part.

**Reply by Naktibalda** [reply]
Site generation will be fine.
There are only 2 place where `master` is mentioned in https://github.com/Codeception/codeception.github.com/blob/master/RoboFile.php

The links on the site won't be broken when repositories are updated, because Github redirects to default branch if master branch is missing.
Example: https://github.com/Codeception/module-laravel/blob/master/src/Codeception/Module/Laravel.php

**Reply by TavoNiievez** [reply]
Hey, Could you please help me for now by renaming the aforementioned branch in the Symfony Module?

**Reply by Naktibalda** [reply]
Renamed.

**Reply by TavoNiievez** [reply]
Thanks :-)
